### PR TITLE
Remove internal usage of Ember.getPath. Instead use Ember.getPathWithoutDeprecation.

### PR DESCRIPTION
**Pull request by ebryn**
@jamesarosen Please review.

### CODE REVIEW

File: packages/ember-handlebars/lib/ext.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ var normalizePath = Ember.Handlebars.normalizePath = function(root, path, data)
   @param {Object} options The template's option hash
 */
 
-Ember.Handlebars.getPath = function(root, path, options) {
+Ember.Handlebars.get = function(root, path, options) {
```
Comment (jamesarosen) [inline packages/ember-handlebars/lib/ext.js:165]: This `get` doesn't obey the `"0.9-dotted-properties"` value.

Comment (ebryn) [inline packages/ember-handlebars/lib/ext.js:165]: Should it have to? It's a new API.

Comment (jamesarosen) [inline packages/ember-handlebars/lib/ext.js:165]: Confirm. No 0.9.8.1 apps should be using this with dotted or non-dotted properties at this point.

**Comment by jamesarosen** [comment]
+1

**Pull request merged**